src/annotator/test/selection-observer-test.js

```diff
@@ -1,39 +1,35 @@
 import { SelectionObserver } from '../selection-observer';
 
-class FakeDocument extends EventTarget {
-  constructor() {
-    super();
-    this.selection = null;
-  }
-
-  getSelection() {
-    return this.selection;
-  }
-}
-
 describe('SelectionObserver', () => {
   let clock;
-  let fakeDocument;
-  let range;
+  let frame;
   let observer;
   let onSelectionChanged;
+  let testDocument;
+
+  function getSelectedRange() {
+    return testDocument.getSelection().getRangeAt(0);
+  }
+
+  before(() => {
+    frame = document.createElement('iframe');
+    document.body.append(frame);
+    testDocument = frame.contentDocument;
+    testDocument.body.innerHTML = 'Some test content';
```

Comment:
Heh, the entire set of changes is worth it just for not using a fake document in these tests :). Glad to see this change.